docs/content/Checkbox.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Checkbox
+status: Alpha
+source: https://github.com/primer/react/blob/main/src/Checklist.tsx
+storybook: '/react/storybook?path=/story/forms-checkbox--default'
+---
+
+import {ComponentChecklist} from '../src/component-checklist'
+
+`Checkbox` is a form component that applies Primer design language to a native HTML checkbox input element.
+
+## Default example
+
+The `Checkbox` component can be used in controlled and uncontrolled modes.
+
+```jsx live
+<>
+  <Box as="form" p={3} pt={0} sx={{display: 'flex', alignItems: 'center'}}>
```

Comment:
`Box` techincally supports both system props and the `sx` prop but it feels a little messy to use them both at the same time. What if we moved all the styling into the `sx` prop in these examples?

```suggestion
  <Box as="form" sx={{p: 3, pt: 0, display: 'flex', alignItems: 'center'}}>
```

Reply:
Fair. That's done now 👍